tests/web/test_metadata.py

```diff
@@ -18,7 +18,7 @@ def test_get_metadata_with_missing_text_search(client):
 
 
 @pytest.mark.genesis
-def test_get_metadata(client, b):
```

Comment:
No need to fix this test, it can be removed in favour of [this](https://github.com/bigchaindb/bigchaindb/pull/2338/files#diff-d67953f9f41851852e17a20de0265318R94)